test/parallel/test-child-process-exec-detached.js

```diff
@@ -0,0 +1,17 @@
+'use strict';
+const common = require('../common');
+const assert = require('assert');
+const exec = require('child_process').exec;
+
+const notDetached = exec('git clone https://github.com/private/private', { timeout: 1000, detached: false });
+const detached = exec('git clone https://github.com/private2/private2', { detached: true, stdio: ['ignore'] });
+
+notDetached.on('exit', () => {
```

Comment:
The `'exit'` handlers should be wrapped in `common.mustCall()`. We should probably add assertions for the exit code and signal as well.